validator/sawtooth_validator/execution/executor.py

```diff
@@ -103,12 +103,21 @@ def _future_done_callback(self, request, result):
         self._in_process_transactions_count.dec()
         req = processor_pb2.TpProcessRequest()
         req.ParseFromString(request)
+        # If raw header bytes were sent then de-serialize to get
+        # transaction family name and version
+        request_header = transaction_pb2.TransactionHeader()
+        if req.header_bytes == b'':
+            # header field must be present
+            request_header = req.header
+        else:
+            # header_bytes must be present
```

Comment:
Maybe change this comment to "Deserialize the header bytes"

Reply:
Done